dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,123 @@
+CWPKI2013E.explanation=The certificate order was created and signed, but ordering the certificate from the certificate authority failed.
+CWPKI2013E.useraction=Review the error message for details on the failure.
+
+CWPKI2014E=CWPKI2014E: The certificate signing request for the ACME certificate authority at the {0} URI was created and signed, but encoding the request failed. The error is {1}.
+CWPKI2014E.explanation=Encoding the certificate signing request failed and a signed certificate cannot be created.
+CWPKI2014E.useraction=Review the error message for details on the failure.
+
+CWPKI2015E=CWPKI2015E: Received an error while requesting a certificate order update from the ACME certificate authority at the {0} URI. The error is {1}.
+CWPKI2015E.explanation=An error occurred while requesting a status update on a certificate order.
+CWPKI2015E.useraction=Review the error message for details on the failure.
+
+CWPKI2016E=CWPKI2016E: The ACME service request for an existing account from the ACME certificate authority at the {0} URI failed The error is {1}.
+CWPKI2016E.explanation=Requested an existing account but it was not found or another error occurred. Changes cannot be made to the account or certificate.
+CWPKI2016E.useraction=Review the error message for details on the failure. Verify that the URI can be successfully accessed by the calling server. Verify that the calling server can receive a response from the certificate authority.
+
+CWPKI2017E=CWPKI2017E: The ACME request for the terms of service from the ACME certificate authority at the {0} URI failed. The error is {1}.
+CWPKI2017E.explanation=Requested the terms of service from the certificate authority, but an error occurred. 
+CWPKI2017E.useraction=Review the error message for details on the failure. Visit the ACME certificate authority website to review the terms of service.
+
+CWPKI2018E=CWPKI2018E: The user account could not be created on the ACME certificate authority at the {0} URI. The error is {1}.
+CWPKI2018E.explanation=The user account creation request failed.
+CWPKI2018E.useraction=Review the error message for details on the failure.
+
+CWPKI2019I=CWPKI2019I: The account URL provided by the ACME certificate authority at the {0} URI is {1}.
+CWPKI2019I.explanation=The account was successfully created.
+CWPKI2019I.useraction=No action is required.
+
+CWPKI2020E=CWPKI2020E: The ACME service could not read the domain key file for the ACME certificate authority account at the {0} URI. The file location is {1}. The error is {2}.
+CWPKI2020E.explanation=The domain key file for the certificate authority account could not be opened. This can occur if file permissions are incorrect or the file does not exist.
+CWPKI2020E.useraction=Review the error message for details on the failure. Verify that the file location is correct and has the correct file permissions.
```

Comment:
does the file location need file permissions? or the service? Or does the location need to grant the service permission to access?